targets/UnrealMarketplacePlugin/source/PlayFab/Source/PlayFabCommon/Public/PlayFabCommonSettings.h.ejs

```diff
@@ -36,7 +37,9 @@ namespace PlayFabCommon
         static FString getURL(const FString& callPath)
         {
             if (serverURL.Len() == 0)
-				serverURL = TEXT("https://") + titleId + productionEnvironmentURL;
+            {
+                serverURL = TEXT("https://") + !verticalName.IsEmpty() ? verticalName : titleId + productionEnvironmentURL;
+            }
 			return serverURL + callPath + TEXT("?sdk=") + versionString;
```

Comment:
I know it was already here... but can you fix the tabbing on this line (tabs -> spaces)
Thanks and sorry.